packages/web/src/common/store/backend/sagas.ts

```diff
@@ -56,6 +56,9 @@ function* awaitReachability() {
 }
 
 export function* setupBackend() {
+  // Optimistically fetch account, then do it again later when we're sure we're connected
+  // This ensures we always get the cached account when starting offline if available
+  yield* put(accountActions.fetchAccount())
```

Comment:
It might be better to make a separate action/saga to get the cached account if it exists. bc `fetchAccount` makes a backend call if the account is not cached (and the saga is `takeEvery`)